src/DIRAC/Resources/Computing/AREXComputingElement.py

```diff
@@ -198,7 +200,7 @@ def _checkSession(self):
             return S_OK()
 
         # Attach the proxy to the session, only if the token is unavailable
-        self.session.cert = Locations.getProxyLocation()
+        self.session.cert = os.environ["X509_USER_PROXY"]
```

Comment:
Why this would be better than the previous implementation?

Reply:
Because `ComputingElement.prepareProxy()` is using `X509_USER_PROXY` to store the CE proxy.
+
In the case you have multiple proxies and you don't call `prepareProxy()`: 
- if `Locations.getProxyLocation()` is called, you might silently use the wrong proxy
- if `os.environ` is called, you will get an error and you will know that you have to clearly define which proxy you want to use 

Any opinion about that?